src/main/java/com/spotinst/sdkjava/example/admin/account/UserManagementUsageExample.java

```diff
@@ -0,0 +1,275 @@
+package com.spotinst.sdkjava.example.admin.account;
+
+import com.spotinst.sdkjava.SpotinstClient;
+import com.spotinst.sdkjava.enums.admin.account.PolicyEffectEnum;
+import com.spotinst.sdkjava.enums.admin.account.PolicyTypeEnum;
+import com.spotinst.sdkjava.model.SpotinstAccountAdminClient;
+import com.spotinst.sdkjava.model.bl.admin.account.*;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class UserManagementUsageExample {
+    private final static String authToken = "auth-token";
+    private final static String accountId = "account-id";
+
+    public static void main(String[] args) {
+        SpotinstAccountAdminClient adminClient = SpotinstClient.getAdminAccountClient(authToken, accountId);
+
+        // User APIs
+        String userId = createUser(adminClient);
+        getUserDetails(adminClient, userId);
+        String programmaticUser = createProgammaticUser(adminClient);
+        getUserDetails(adminClient, programmaticUser);
+        getOrganizationUsers(adminClient);
+        updateGroupsofUsers(adminClient, userId);
+        updateUserDirectPolicies(adminClient, userId);
```

Comment:
deleteUser API is not there?

Reply:
Yes. DeleteUser is not supported through API with New/Old UMS